app/models/request.rb

```diff
@@ -3,16 +3,14 @@ class Request < ActiveRecord::Base
   aasm :column => :status do 
     state :request_pick_up, :initial => true
     state :valet_pick_up
-    state :car_pick_up
     state :in_transit_pick_up
     state :car_parked
     state :request_drop_off
     state :valet_drop_off
-    state :in_transit_drop_off
-    state :car_drop_off
+    state :e
```

Comment:
I have no idea, must have hit E accidentally.  suppose to be in_transit_drop_off